### PR TITLE
Feat/yel7490a

### PR DESCRIPTION
**Pull request by s-arunachalam**

### CODE REVIEW

File: app/monitor/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3-alpine
+FROM python:3.10.0-alpine
```
Comment (ianrelf) [inline app/monitor/Dockerfile:1]: Why change from `3-alpine` to `3.10.0-alpine`?
Doesn't `3-alpine` link to the latest release?

File: deployment.yml [hunk 2]
```diff
@@ -137,7 +137,7 @@ spec:
           - name: HOST
             value: '10.3.0.12'
           - name: PORT
-            value: '8088'
+            value: '8081'
```
Comment (ianrelf) [inline deployment.yml:140]: Why does this port number need to be changed?

File: app/monitor/Dockerfile (hunk 1 above)
Comment (s-arunachalam) [reply app/monitor/Dockerfile:1]: this is a comment from Ajilesh, we want to keep in sync with CDLZ who have updated to 3.10.0-alpine. :-)

File: deployment.yml (hunk 2 above)
Comment (s-arunachalam) [reply deployment.yml:140]: Port number at 8088 seems to be blocking deployment, historically Ola has changed to 8081 from 80888 and put back the port.

**Pull request merged**